logto-io/logto
# chore: enable Node 18

## Summary

- add node 18 to engines
- add node 18 to test workflows
- update cli node version check accordingly
- remove pptr Chromium cache, manually install before integration test since only integration test needs it


## Testing


- CI 
- locally with Node 18
- tested CLI

<img width="606" alt="image" src="https://user-images.githubusercontent.com/14722250/200131558-e4faa06c-b33b-4f75-9690-c3909270a68d.png">
